lib/jsdom/living/nodes/HTMLCanvasElement-impl.js

```diff
@@ -69,6 +73,7 @@ class HTMLCanvasElementImpl extends HTMLElementImpl {
     const canvas = this._getCanvas();
     if (canvas) {
       let stream;
+      // TODO: use toBuffer instead: faster and less code.
```

Comment:
Is this something that could be done easily? If so I wonder if it might be easier to just do so in this PR?